lua/damagelogs/config/config.lua

```diff
@@ -88,7 +89,7 @@ Damagelog.UseWorkshop = true
 -- Force a language - When empty use user-defined language
 Damagelog.ForcedLanguage = ""
 -- Allow reports even with no staff online
-Damagelog.NoStaffReports = false
+Damagelog.NoStaffReports = true
```

Comment:
Was this change intentional?
I try to leave the default config the way it is to avoid causing confusing when people upgrade.

Reply:
it wasn't